contracts/Core/BlockManager.sol

```diff
@@ -147,6 +147,22 @@ contract BlockManager is Initializable, ACL, BlockStorage, StateManager, IBlockM
         randomNoProvider.provideSecret(epoch - 1, voteManager.getRandaoHash());
     }
 
+    function disputeBiggestInfluenceProposed(
+        uint32 epoch,
+        uint8 blockIndex,
+        uint32 correctBiggestInfluencerId
+    ) external initialized checkEpochAndState(State.Dispute, epoch, parameters.epochLength()) returns (uint32) {
+        uint8 blockId = sortedProposedBlockIds[epoch][blockIndex];
+        require(proposedBlocks[epoch][blockId].valid, "Block already has been disputed");
+        uint256 correctBiggestInfluence = stakeManager.getInfluence(correctBiggestInfluencerId);
+        require(correctBiggestInfluence > proposedBlocks[epoch][blockId].biggestInfluence, "Invalid dispute : Influence");
+
+        proposedBlocks[epoch][blockId].valid = false;
+        _findNextBlockToBeConfirmed(epoch, blockIndex);
+        uint32 proposerId = proposedBlocks[epoch][blockId].proposerId;
```

Comment:
if we see these 3 lines are common as well, here
160, 162,163
and then in finalise dispute 188, 190, 191

we can move this in _findNextBlockToBeConfirmed only, just not moved for readability reasons
Let me know if you think otherwise, we can move to save some storage cost